app/models/ability.rb

```diff
@@ -125,6 +125,7 @@ def public_ability
 
     # Required in stockit
     can [:index, :create], Location
+    can :create, Package
```

Comment:
Can you annotate this with #TODO to make a note to tighten this up when we add authentication... this is very open right now!